pallets/stableswap/src/lib.rs

```diff
@@ -878,6 +880,113 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(10)]
+		#[pallet::weight(<T as Config>::WeightInfo::remove_liquidity())]
```

Comment:
no need.we benchmark for max assets in pool. which is 5 atm, so wont be big difference. 

can create an issue to rework but it would be needed for all other calls too